grc/core/FlowGraph.py

```diff
@@ -113,6 +113,8 @@ def get_snippets_dict(self, section=None):
             d['section'] = sect
             d['priority'] = snip.params['priority'].value
             d['lines'] = snip.params['code'].value.splitlines()
+            if not len(d['lines']):
```

Comment:
Same problem if there's just a comment. No problem at all if there's just a literal. All kinds of ways to make bad code with a snippet. Would a warning somewhere be better than silently ignoring? Can't think of a best way. In any case, this gets rid of the most obvious case.

Reply:
Good point on the warning - silently failing is not usually a good thing